app/src/main/res/values/strings.xml

```diff
@@ -259,6 +257,10 @@
     <string name="carrier_detector_tool">Carrier detector tool</string>
     <string name="carrier_detector_tool_description">This tool tries to find possible carriers for given tracking code. If tool gives options, you can select one via clicking carrier image.</string>
     <string name="reference_to_database_is_null_cannot_add_items">Reference to database is missing, cannot add items!</string>
+    <string name="main_menu_taking_backup_success">Taking backup was successful. File can be found from: </string>
```

Comment:
"Backup was succesful. Backup file can be found from: "

Reply:
Ok